pkgdb2client/cli.py

```diff
@@ -51,14 +51,14 @@ class ActionError(Exception):
     pass
 
 
-def __get_namespace(args):
+def __get_namespace(args, kw='package'):
     ''' Return the namespace of the package as specified as an argument
     unless it was specified in the package name.
     '''
     namespace = args.namespace
-    if hasattr(args, 'package'):
-        if args.package and '/' in args.package:
-            namespace = args.package.split('/', 1)[0]
+    if hasattr(args, kw):
```

Comment:
A small suggestion here: I have recently stopped using `hasattr` in favor of `try-except` after reading this - https://hynek.me/articles/hasattr/